contributors.md

```diff
@@ -2,90 +2,107 @@
-In the latter case, an R-multiverse moderator will manually review your pull request and contact you if there are questions.
-Please see our [review policy](review.md) for more information about the process.
+## Topics
 
-## How the R-multiverse universe is built
+R-multiverse has a ["topics" system](https://r-multiverse.org/topics/index.html) for organizing packages into subject matter areas.
+Each topic has a page that lists the packages in its area ([example](https://r-multiverse.org/topics/bayesian.html)).
+To contribute a package to a topic, simply list the topic URL in the `DESCRIPTION` file ([example](https://github.com/ropensci/stantargets/blob/db7d119ea0599eac3ce01a42bee27c9908754943/DESCRIPTION#L22)).
+and then create a new GitHub/GitLab release.
+A bot periodically scans [Community](community.md) and updates the list of packages in each topic.
+
+To contribute a topic, submit a pull request to <https://github.com/r-multiverse/topics> to add a new text file (with no file extension) whose name is the topic name and whose contents describe the topic.
+The bot generates HTML pages from these text files.
+
+## Badges
+
+[<img src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2Fmirai&query=%24.Version&label=r-multiverse" alt="R-multiverse status" />](https://community.r-multiverse.org/mirai)
+
+To add a dynamic 'R-multiverse' badge for package readme files, like the one above, copy the following markdown snippet, replacing 'pkgNAME' with the actual package name in both places it appears:
+
+```md
+[![R-multiverse status](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2FpkgNAME&query=%24.Version&label=r-multiverse)](https://community.r-multiverse.org/pkgNAME)
+```
 
-Periodically, a GitHub Actions workflow collects all the packages and URLs from the [`packages`](https://github.com/r-multiverse/contributions/tree/main/packages) folder and builds the `packages.json` file for the universe at <https://github.com/r-multiverse/community>.
-<https://community.r-multiverse.org> periodically refreshes to include the latest release of each package in `packages.json`.
+## Deregistration
 
-## Ownership and attribution
+If it becomes necessary to deregister a package from R-multiverse, you may submit a
+[pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+to remove the [package listing](https://github.com/r-multiverse/contributions/tree/main/packages).
+To prevent a package of the same name from registering in R-multiverse,
+you may contribute a listing with free-form text (not a URL or JSON list).
+Packages with missing or free-form listings are removed from [Community](community.md), and the subsequent [Production](production.md) snapshot will not include them. 
```

Comment:
even if it's sort of obvious, add that the package is never removed from past snapshots?

Reply:
Added a note in 6f5a52b08e0c7a8bc2d3566d94afdf0c9721b77f.